jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -170,44 +181,63 @@ public static Result run(String inPaths, String pkgName, String outPath, boolean
                       .toString()
                       .equals("Primordial")) {
                 Preconditions.checkNotNull(mtd, "method not found");
-                // Skip methods by looking at bytecode
-                try {
-                  if (CodeScanner.getFieldsRead(mtd).isEmpty()
-                      && CodeScanner.getFieldsWritten(mtd).isEmpty()
-                      && CodeScanner.getCallSites(mtd).isEmpty()) {
-                    continue;
+                DefinitelyDerefedParams analysisDriver = null;
+                String sign = "";
+                // Parameter analysis
+                if (mtd.getNumberOfParameters() > (mtd.isStatic() ? 0 : 1)) {
```

Comment:
The "no parameters" and "doesn't contain relevant instructions" tests should only be used to elide parameter analysis. Decoupled this logic from that used to elide return value analysis.